ui/components/passed.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+class PassedTest extends React.Component {
+  render() {
+    const baseline = this.props.paths.baseline;
+
+    return <div className="diffs">
+      <img className="baseline"
+           src={baseline}
+           key={baseline} />
```

Comment:
No need for `key`. The attribute is only useful when elements of the same type can switch places in the output.